### DISCUSSION TITLE
Run renterd and hostd on the same consensus

### DISCUSSION THREAD

**Discussion by hackschnitzel09** [SiaFoundation/hostd#205] [Questions & Support] [answered]
Hi, Id like to know if it is possible to have both running but only need one consensus on the system or even connect to an existing running node?

**Comment by n8mgr** [comment, marked as answer]
It is possible, but would require a wrapper to connect the two to the same consensus db. Utreexo will make this problem obsolete.

**Comment by npelov** [comment]
On linux with ZFS filesystem you can make dataset for the consensus db with dedup=on. Technically tehre will be two databases but shouldn't take much more space
